server/controllers/fetchData/topics.js

```diff
@@ -0,0 +1,819 @@
+    title: 'Create a Column Gap Using grid-column-gap',
+  },
+  {
+    id: '5a9036ee38fddaf9a66b5d36',
+    title: 'Create a Row Gap using grid-row-gap',
+  },
+  {
+    id: '5a9036ee38fddaf9a66b5d37',
+    title: 'Add Gaps Faster with grid-gap',
+  },
+  {
+    id: '5a90372638fddaf9a66b5d38',
+    title: 'Use grid-column to Control Spacing',
+  },
+  {
+    id: '5a90373638fddaf9a66b5d39',
+    title: 'Use grid-row to Control Spacing',
+  },
+  {
+    id: '5a90374338fddaf9a66b5d3a',
+    title: 'Align an Item Horizontally using justify-self',
+  },
+  {
+    id: '5a90375238fddaf9a66b5d3b',
+    title: 'Align an Item Vertically using align-self',
+  },
+  {
+    id: '5a90376038fddaf9a66b5d3c',
+    title: 'Align All Items Horizontally using justify-items',
+  },
+  {
+    id: '5a94fdf869fb03452672e45b',
+    title: 'Align All Items Vertically using align-items',
+  },
+  {
+    id: '5a94fe0569fb03452672e45c',
+    title: 'Divide the Grid Into an Area Template',
+  },
+  {
+    id: '5a94fe1369fb03452672e45d',
+    title: 'Place Items in Grid Areas Using the grid-area Property',
+  },
+  {
+    id: '5a94fe2669fb03452672e45e',
+    title: 'Use grid-area Without Creating an Areas Template',
+  },
+  {
+    id: '5a94fe3669fb03452672e45f',
+    title: 'Reduce Repetition Using the repeat Function',
+  },
+  {
+    id: '5a94fe4469fb03452672e460',
+    title: 'Limit Item Size Using the minmax Function',
+  },
+  {
+    id: '5a94fe5469fb03452672e461',
+    title: 'Create Flexible Layouts Using auto-fill',
+  },
+  {
+    id: '5a94fe6269fb03452672e462',
+    title: 'Create Flexible Layouts Using auto-fit',
+  },
+  {
+    id: '5a94fe7769fb03452672e463',
+    title: 'Use Media Queries to Create Responsive Layouts',
+  },
+  {
+    id: '5a94fe8569fb03452672e464',
+    title: 'Create Grids within Grids',
+  },
+  {
+    id: 'bd7158d8c442eddfaeb5bd18',
+    title: 'Build a Tribute Page',
+  },
+  {
+    id: '587d78af367417b2b2512b03',
+    title: 'Build a Survey Form',
+  },
+  {
+    id: '587d78af367417b2b2512b04',
+    title: 'Build a Product Landing Page',
+  },
+  {
+    id: '587d78b0367417b2b2512b05',
+    title: 'Build a Technical Documentation Page',
+  },
+  {
+    id: 'bd7158d8c242eddfaeb5bd13',
+    title: 'Build a Personal Portfolio Webpage',
+  },
+  {
+    id: 'bd7123c9c441eddfaeb4bdef',
+    title: 'Comment Your JavaScript Code',
+  },
+  {
+    id: 'bd7123c9c443eddfaeb5bdef',
+    title: 'Declare JavaScript Variables',
+  },
+  {
+    id: '56533eb9ac21ba0edf2244a8',
+    title: 'Storing Values with the Assignment Operator',
+  },
+  {
+    id: '56533eb9ac21ba0edf2244a9',
+    title: 'Initializing Variables with the Assignment Operator',
+  },
+  {
+    id: '56533eb9ac21ba0edf2244aa',
+    title: 'Understanding Uninitialized Variables',
+  },
+  {
+    id: '56533eb9ac21ba0edf2244ab',
+    title: 'Understanding Case Sensitivity in Variables',
+  },
+];
+
+const topicId = [];
+
+topics.map(({ id }) => {
+  topicId.push(id);
+});
```

Comment:
I prefer to put these line in separate file called `getTopicsId.js` and let this file only for topics, cause its too long.